unittests/producer_schedule_tests.cpp

```diff
@@ -11,187 +11,122 @@ using mvo = fc::mutable_variant_object;
 
 BOOST_AUTO_TEST_SUITE(producer_schedule_tests)
 
-   // Calculate expected producer given the schedule and slot number
-   account_name get_expected_producer(const vector<producer_authority>& schedule, const uint64_t slot) {
-      const auto& index = (slot % (schedule.size() * config::producer_repetitions)) / config::producer_repetitions;
-      return schedule.at(index).producer_name;
-   };
+namespace {
+
+// Calculate expected producer given the schedule and slot number
+account_name get_expected_producer(const vector<producer_authority>& schedule, block_timestamp_type t) {
+   const auto& index = (t.slot % (schedule.size() * config::producer_repetitions)) / config::producer_repetitions;
+   return schedule.at(index).producer_name;
+};
+
+// Check if two schedule is equal
+bool is_schedule_equal(const vector<producer_authority>& first, const vector<producer_authority>& second) {
```

Comment:
How is this implementation different from `return first == second;`?

Reply:
lol, it is not different. I'll remove it.